client/src/app/__tests__/AppSpec.js

```diff
@@ -308,6 +308,24 @@ describe('<App>', function() {
       expect(activeTab).to.eql(tabs[ tabs.length - 1 ]);
     });
 
+    it('should include labelSuffix for Camunda Platform 8', function() {
```

Comment:
Is `diagramCreation` really the right place? Why not just a new `describe('#_getNewFileitems')`?